x-pack/packages/kbn-elastic-assistant/impl/assistant/api.tsx

```diff
@@ -8,7 +8,9 @@
 import { OpenAiProviderType } from '@kbn/stack-connectors-plugin/public/common';
 
 import { HttpSetup, IHttpFetchError } from '@kbn/core-http-browser';
-
+// TODO: Why do i get this error here? its imported in other places without issue
+// eslint-disable-next-line import/no-nodejs-modules
+import { IncomingMessage } from 'http';
```

Comment:
I think it's because it's coming from within a client/UI only package? You should be able to just import the type to get around it:

```suggestion
import type { IncomingMessage } from 'http';
```

Reply:
looks like i still get it 😢 
<img width="652" alt="Screenshot 2023-10-25 at 5 07 05 PM" src="https://github.com/elastic/kibana/assets/6935300/ef5eed02-3bcc-4494-bc53-cacca9b62bbb">

Reply:
May just want to remove it outright as the downstream reader type appears to be fine?


<img width="701" alt="image" src="https://github.com/elastic/kibana/assets/2946766/797ab0ac-49f9-4ba4-933c-f4c2fbea7c88">